app/src/main/java/me/saket/dank/ui/webview/ChromeCustomTabsForwardingActivity.java

```diff
@@ -40,8 +42,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     CustomTabsIntent customTabsIntent = builder.build();
 
     String url = getIntent().getStringExtra(KEY_URL);
-    customTabsIntent.launchUrl(this, Uri.parse(url));
-
-    finish();
+    try {
+      customTabsIntent.launchUrl(this, Uri.parse(url));
+      finish();
```

Comment:
Are you sure that `finish` should be inside of try block here? The activity appears to have no user interface and having it stay on top could be problematic.